src/lib/dataEntities/networks.ts

```diff
@@ -64,6 +64,7 @@ export interface EthBridge {
   sequencerInbox: string
   outbox: string
   rollup: string
+  classicOutbox: string
```

Comment:
would be good to make `classicOutbox` optional instead of using `0x0000000000000000000000000000000000000000`

Reply:
updated